utils/products.js

```diff
@@ -1,152 +1,664 @@
     githubRepo: [
       "https://github.com/eclipse-tractusx/daps-registration-service",
       "https://github.com/eclipse-tractusx/daps-helm-chart",
       "https://github.com/eclipse-tractusx/sd-factory",
-      "https://github.com/eclipse-tractusx/autosetup-backend"
     ],
     committers: [
-    "https://github.com/wjost"
+      "https://github.com/tomaszbarwicki",
+      "https://github.com/almadigabor",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request IDS Essential Services Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request IDS Essential Services Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
   },
 
   {
-    productName: "Generic Data Services",
-    productDescription: "The project provides an easy-to-use service which enables small and medium sized companies to provide their data in the Catena-X dataspace via an EDC (e.g., CSV-Upload).",
+    productName: "Item Relationship Service",
+    productDescription:
+      "The project provides a service for ad-hoc data chains across n-tier values chains for different use cases based on the EDC.",
     githubRepo: [
-      "https://github.com/eclipse-tractusx/dft-backend",
-      "https://github.com/eclipse-tractusx/dft-frontend"
+      "https://github.com/eclipse-tractusx/item-relationship-service",
     ],
     committers: [
-      "https://github.com/wjost"
+      "https://github.com/ds-jhartmann",
+      "https://github.com/mkanal",
+      "https://github.com/ds-lcapellino",
+      "https://github.com/ds-mwesener",
+      "https://github.com/ds-jkreutzfeld",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Semantic Layer & Digital Twin Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Item Relationship Service Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
   },
-
   {
-    productName: "Item Relationship Service",
-    productDescription: "The project provides a service for ad-hoc data chains across n-tier values chains for different use cases based on the EDC.",
+    productName: "knowledge-agents",
+    productDescription:
+      "Reference Implementations of CX-0084 (Federated Queries In Data Spaces)",
+    githubRepo: ["https://github.com/eclipse-tractusx/knowledge-agents"],
+    committers: ["https://github.com/almadigabor"],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request knowledge-agents Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "knowledge-agents-aas-bridge",
+    productDescription: "TBD",
     githubRepo: [
-      "https://github.com/eclipse-tractusx/item-relationship-service",
+      "https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge",
+    ],
+    committers: ["https://github.com/almadigabor"],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request knowledge-agents-aas-bridge Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "knowledge-agents-edc",
```

Comment:
Are EDC extensions a specific product?

Reply:
I'd say this is also part of product "Knowledge Agents". I would just add the repo url to the product above. What do you think @drcgjung?